zulipterminal/ui_tools/views.py

```diff
@@ -1211,6 +1213,46 @@ def __init__(self, controller: Any, title: str) -> None:
         super().__init__(controller, widgets, "HELP", popup_width, title)
 
 
+class MarkdownHelpView(PopUpView):
+    def __init__(self, controller: Any, title: str, server_url: str) -> None:
+        raw_menu_content = []  # to calculate table dimensions
+        rendered_menu_content = []  # to display rendered content in table
+
+        for element in MARKDOWN_ELEMENTS:
+            content = element["content"]
+
+            # Currently we only re-render user names
+            if element["re_render_required"]:
+                content = controller.model.user_full_name
+
+            wrapped_html_element = format_string([content], element["html_format"])[0]
+            wrapped_raw_content = format_string([content], element["raw_format"])[0]
```

Comment:
See my comment on the stream, but I think this (and the dict format) can be simplified.